Tasks/ANT/package.json

```diff
@@ -1,12 +1,14 @@
 {
-  "name": "Ant",
+  "name": "Agent.Tasks.Ant",
```

Comment:
fixed this. apparently, this is how other maven/gradle tasks named. Anyway changing back to original one